src/api/FactsController.js

```diff
@@ -0,0 +1,14 @@
+export async function retrieveAnimalFact() {
+  const animalTypes = ['dog', 'cat'];
+  const randomAnimalType = randomIntFromInterval(0, animalTypes.length - 1);
```

Comment:
it is not randomAnimalType but just random number - rename variable or take a random item from an array